packages/migrate/src/emitEvent.ts

```diff
@@ -0,0 +1,7 @@
+import type Config from "@truffle/config";
```

Comment:
You use `import type` here but not on other file for the same Config. Is there anything special here?

Reply:
Yes, in this file the class `Config` is not used. The only thing used here is the type `Config`. This is useful because the types do not make it into the compiled, distributed files.